src/sharding/Shard.js

```diff
@@ -99,29 +90,80 @@ class Shard extends EventEmitter {
     this._exitListener = this._handleExit.bind(this, undefined);
   }
 
+  /**
+   * Process of the shard (if {@link ShardingManager#mode} is `process`)
```

Comment:
These are not accurate... This doesn't return just the shard's actual forked process, but also the class that spawns it... Honestly, This should probably stay as a property